app/views/idv/finance/new.html.slim

```diff
@@ -13,3 +13,5 @@ p.italic Please provide only one of the following:
   .mt4.mb1
     = link_to 'Cancel', idv_cancel_path, class: 'btn btn-primary bg-gray mr2'
     = f.button :submit, 'Continue', class: 'btn btn-primary'
+
+= page_before_unload_js
```

Comment:
It actually does. See: https://github.com/teampoltergeist/poltergeist/blob/master/lib/capybara/poltergeist/driver.rb#L154-L176

Unfortunately, the documentation is sparse, so I've been trying to figure out how to make this work by reading the code.

I was able to close the current window like this:
```ruby
page.driver.close_window(page.driver.current_window_handle)
```
Now I'm trying to figure out how to interact with the alert.